flow/project.py

```diff
@@ -4112,6 +4112,14 @@ def _main_run(self, args):
 
     def _main_script(self, args):
         "Generate a script for the execution of operations."
+        print(
+            "WARNING: "
+            "The script argument is deprecated as of 0.12 "
+            "and will be removed in 0.14. "
+            "Use `python project.py submit --pretend` instead.",
```

Comment:
Not every user will have "python project.py." I'll commit this.
```suggestion
            "Use \"submit --pretend\" instead.",
```